epub34/rs/index.html

```diff
@@ -2460,17 +2457,17 @@ <h3>Recommendations</h3>
 				<p>If a reading system allows users to store persistent data, especially personally identifiable
 					information, it SHOULD treat that data as sensitive and not allow access to it by third parties.</p>
 
-				<p>It is understood that the collection of some user data may be required for the sale, delivery, and
+				<p>It is understood that the collection of some user data is often required for the sale, delivery, and
 					operation of an [=EPUB publication=], particularly on platforms where the sale of an EPUB
 					publication and the method of reading it are connected. In these cases, the reading system SHOULD
-					identify the data being collected, how it is used, and allow for user opt-outs (retailers may choose
-					to inform users by other means, however, such as when a user creates an account on their web site).
+					identify the data being collected, how it is used, and allow for user opt-outs (retailers could
+					choose to inform users by other means, such as when a user creates an account on their web site).
 					Anonymization of any collected data is RECOMMENDED for the privacy and the security of the user and
 					reading system.</p>
 
-				<p>It is also understood that user data may be required or helpful for some reading system affordances.
-					In these cases, anonymization is also RECOMMENDED. Reading systems also SHOULD inform users of what
-					data is needed, what it is to be used for, and to provide methods to opt-out.</p>
+				<p>It is also understood that user data could be required or helpful for some reading system
+					affordances. In these cases, anonymization is also RECOMMENDED. Reading systems also SHOULD inform
+					users of what data is needed, what it is to be used for, and to provide methods to opt-out.</p>
```

Comment:
```suggestion
					affordances. In these cases, anonymization is also RECOMMENDED. Reading systems SHOULD also inform
					users of what data is needed, what it is to be used for, and to provide methods to opt-out.</p>
```

Isn't that a better word order? (I note that the order was similar in the original, too.)